docs/capabilities/app-client.md

```diff
@@ -108,9 +108,9 @@ The create method is a wrapper over the `appCreate` (bare calls) and `appCreateM
 
 ```typescript
 // Use no-argument bare-call
-const { result, app } = factory.create()
+const { result, appClient } = factory.send.bare.create()
```

Comment:
@SilentRhetoric even though it adds a little bit of verbosity we made the decision that every time a transaction is sent to the network you will see the word `send` (apart from the `deploy` call, but that's a special case since it may or may not send transactions and is a higher-order function).

As part of this we renamed `execute` to `send` (with a backwards compatible `@deprecate`) in `AlgoKitComposer` and renamed `ExecuteParams` to `SendParams`.